monitor.py

```diff
@@ -0,0 +1,65 @@
+""" Script to monitor what the Rover is seeing
```

Comment:
Folks new to ROS may get confused that this script is outside the ROS application. Perhaps we can rename it to something like: s3_results_monitor.py... Then describe it as a "tool that folks can use to check what results are appearing in s3".

Reply:
yes, will rename, comment in the script and describe in the workshop material